### PR TITLE
Fix for #9244

### PR DESCRIPTION
**Pull request by dbacinski**

### CODE REVIEW

File: xxx/build.gradle [hunk 1]
```diff
@@ -2,7 +2,6 @@ apply plugin: 'kotlin'
 apply plugin: 'kotlin-spring'
 apply plugin: 'java'
 apply plugin: 'eclipse'
-apply plugin: 'org.springframework.boot'
```
Comment (dbacinski) [inline xxx/build.gradle:5]: thanks to this bootJar task is removed and jar task is back active

File: xxx/build.gradle [hunk 2]
```diff
@@ -21,8 +20,8 @@ repositories {
 }
 
 dependencies {
-    compile('org.springframework.boot:spring-boot-starter-webflux')
-    testCompile('org.springframework.boot:spring-boot-starter-test')
+    compile("org.springframework.boot:spring-boot-starter-webflux:${springBootVersion}")
+    testCompile("org.springframework.boot:spring-boot-starter-test:${springBootVersion}")
```
Comment (dbacinski) [inline xxx/build.gradle:24]: `dependency-management` plugin do not fill versions without `spring boot` plugin so I have to define it explicitly.

Comment (snicoll) [reply xxx/build.gradle:24]: That's not right, it does. Maybe @wilkinsona knows what the problem is on your project.

**Comment by wilkinsona** [comment]
You should import Spring Boot's bom:

```groovy
dependencyManagement {
    imports {
        mavenBom "org.springframework.boot:spring-boot-dependencies:${springBootVersion}"
    }
}
```

**Comment by dbacinski** [comment]
@wilkinsona Thanks it now works https://github.com/dbacinski/spring-boot-issue-9244/pull/2/files

**Comment by wilkinsona** [comment]
Excellent

**Pull request merged**